### ISSUE TITLE
Plugin install fails on Unity Package Manager

### ISSUE THREAD

**Issue by kshoji** [kshoji/Nearby-Connections-MIDI-for-Unity#1]
```
  Package jp.kshoji.unity.nearby.midi@ssh://git@github.com/kshoji/Nearby-Connections-MIDI-for-Unity.git has invalid dependencies or related test packages:
    jp.kshoji.unity.nearby (dependency): Version 'ssh://git@github.com/kshoji/Nearby-Connections-for-Unity.git' is invalid. Expected a 'SemVer' compatible value.
```

Specify the `SemVer` version to `package.json` file.
https://docs.unity3d.com/Manual/upm-git.html#revision

**Comment by kshoji** [comment]
Maybe this is the same problem.
https://issuetracker.unity3d.com/issues/the-error-invalid-expected-a-semver-compatible-value-is-thrown-when-adding-a-dependency-in-the-inspector-with-letters-in-the-version-name

**Comment by kshoji** [comment]
Same issue happened with
https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib
```
[Package Manager Window] Error adding package: https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib.git.
Unable to add package [https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib.git]:
  Package com.unity.innovationgroup.filmlib@https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib.git has invalid dependencies or related test packages:
    com.newtonsoft.json (dependency): Version 'https://github.com/PixelWizards/com.newtonsoft.json.git' is invalid. Expected a 'SemVer' compatible value.
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()
```

This repo's issue has reported:
https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib/issues/4

**Comment by kshoji** [comment]
Same issue happened with
https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib
```
[Package Manager Window] Error adding package: https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib.git.
Unable to add package [https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib.git]:
  Package com.unity.innovationgroup.filmlib@https://github.com/Unity-Technologies/com.unity.innovationgroup.filmlib.git has invalid dependencies or related test packages:
    com.newtonsoft.json (dependency): Version 'https://github.com/PixelWizards/com.newtonsoft.json.git' is invalid. Expected a 'SemVer' compatible value.
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()
```

**Comment by kshoji** [comment]
Remove the dependency and document about it.

**Comment by kshoji** [comment]
Added a fix at #2. I'll leave it open since the problem still exists.

**Comment by kshoji** [comment]
Currently, this seems to be a specification of Unity Package Manager.
https://forum.unity.com/threads/unable-to-set-git-dependency-in-package-json.1385907/
